src/areyouok_telegram/data/models/command_usage.py

```
@@ -0,0 +1,70 @@
+            stmt = pg_insert(cls).values(
+                command=command,
+                chat_id=str(chat_id),
+                session_id=session_id,
+                timestamp=now,
+            )
+
+            result = await db_conn.execute(stmt)
+
+        # Catch exceptions here to avoid breaking application flow
+        # This is a best-effort logging, so we log the exception but don't raise it
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)